src/karts/kart.cpp

```diff
@@ -2107,11 +2137,11 @@ void Kart::updatePhysics(float dt)
     // Only apply if near ground instead of purely based on speed avoiding
     // the "parachute on top" look.
     const Vec3 &v = m_body->getLinearVelocity();
-    if(/*isNearGround() &&*/ v.getY() < - m_kart_properties->getSuspensionTravel()*60)
+    if(/*isNearGround() &&*/ v.getY() < - m_kart_properties->getSuspensionTravel() * 0.01f * 60)
```

Comment:
That actually is commented just 
just before the shown snippet ;)

I expect a conflict there, originally the suspension travel was in cm, and I changed the units a while ago to make it all consistently use m.